_maps/map_files/OmegaStation/OmegaStation.dmm

```diff
@@ -22738,7 +22611,9 @@
 /turf/open/floor/engine,
 /area/engine/engineering)
 "aOa" = (
-/obj/machinery/power/supermatter_crystal/shard/engine,
+/obj/machinery/nuclearbomb/selfdestruct{
+	r_code = "11111"
```

Comment:
```suggestion
	r_code = "voreisbad"
```

Reply:
bitch i CANNOT put numbers into the nuke code

Reply:
i meant letters

Reply:
Code it so you can, cant be that hard, right?